tests/fast-integration/image/Dockerfile

```diff
@@ -1,14 +1,16 @@
-FROM alpine:3.14.6
+FROM alpine:3.16.2
 
 ENV FIT_MAKE_TARGET=ci-skr
 
 SHELL ["/bin/ash", "-o", "pipefail", "-c"]
 
+RUN apk update; apk upgrade
+
 RUN apk --no-cache add \
     build-base \
     git \
     curl \
-    nodejs=14.19.0-r0 \
+    nodejs=16.16.0-r0 \
```

Comment:
For reviewer:
Updated the version, because there was a error:
```
[0005] | [91mERROR: unable to select packages:
[0005] | [0m  nodejs-14.20.0-r0:
```

Reply:
Do we need to install a newer version of 14.xx instead ?
The [version for the prow tests](https://github.com/kyma-project/test-infra/blob/main/prow/images/buildpack-node/Dockerfile#L15) is still on node js 12 🤔

Reply:
The latest v14.x version was also giving errors while installing. I don't know why prow is still using v14, but locally I am using v16 for a while and the fi-tests are okay. Also, there seems to be no [breaking changes in v16](https://nodejs.org/en/blog/release/v16.0.0/), which could effect us in FI tests.

Reply:
Sorry wrong dockerfile, [this is the correct one](https://github.com/kyma-project/test-infra/blob/main/prow/images/kyma-integration/Dockerfile#L100) which is using version 14

Reply:
What about we update the nodejs version for prow as well ? :)

Reply:
let do it in a separate PR, maybe we can use [official nodejs image](https://hub.docker.com/layers/node/library/node/14.20.0-alpine3.16/images/sha256-9e9730aa7994a7d4deb26e4d5784d26c0d2c01d28de7833872cac6b8bd23a263?context=explore).